packages/workbox-background-sync/test/browser/background-sync-queue-plugin.js

```diff
@@ -12,19 +12,19 @@
  */
 
 /* eslint-env mocha, browser */
-/* global chai, workbox */
+/* global expect, workbox */
 
 'use strict';
 
 describe('background-sync-queue-plugin test', () => {
```

Comment:
You probably shouldn't have the word "test" here. `describe` is supposed to describe the functionality of the module being tested, not the test itself.